### PR TITLE
feat: add category 6 for 'dokumen kesehatan STRTTK' docs

### PR DESCRIPTION
**Pull request by indraprasetya154**
## Overview
- feat: add category 6 for 'dokumen kesehatan STRTTK' docs

cc: @jabardigitalservice/jds-backend 

## Related Task
https://app.clickup.com/t/2vjhcrc

## Preview File Generated
https://app.clickup.com/t/2vj984w?comment=990880109

## Evidence
title: feat: add category 6 for 'dokumen kesehatan STRTTK' docs
project: SIKD
participants: @indraprasetya154 @azophy @samudra-ajri @fajarhikmal214

### CODE REVIEW

File: src/controllers/category/6.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export function drawTemplate6(req, pages, arialRegularFont, QRCodeImagePng, QRCodeImageLogo, footerUrl) {
+    for (const page of pages) {
+        let { width, height } = page.getSize();
+
+        // Convert from point (pt) to centimeter (cm)
+        width = Math.ceil(convert(width, 'pt', 'cm'));
+        height = Math.ceil(convert(height, 'pt', 'cm'));
+        // this document only potrait mode
+        drawText6(req, page, arialRegularFont, QRCodeImagePng, QRCodeImageLogo, footerUrl)
```
Comment (fajarhikmal214) [inline src/controllers/category/6.js:12]: I want to ask, this has not been implemented for the possibility of dynamic paper orientation (landscape & portrait) ? or is it not needed ?

Comment (indraprasetya154) [reply src/controllers/category/6.js:12]: Nice question.. i already ask to PM and the document will be not have multiple orientation

Comment (fajarhikmal214) [reply src/controllers/category/6.js:12]: All right then. Thanks.

**Review by fajarhikmal214 — APPROVED** [review]

File: src/controllers/category/6.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import convert from 'convert-length';
+import { drawText6 } from '../../services/draw-text/6.js'
+
+export function drawTemplate6(req, pages, arialRegularFont, QRCodeImagePng, QRCodeImageLogo, footerUrl) {
+    for (const page of pages) {
```
Comment (samudra-ajri) [inline src/controllers/category/6.js:5]: This looping method is great!
I usually using the `foreach` loop method to handle an array value looping. However, a momoent ago, I just relised that this `for` loop method alternative is 95% faster than the `foreach`. 👍

**Review by samudra-ajri — APPROVED** [review]

File: src/controllers/category/6.js (hunk 2 above)
Comment (indraprasetya154) [reply src/controllers/category/6.js:5]: Yes man, this one is good explain https://stackoverflow.com/questions/50844095/should-one-use-for-of-or-foreach-when-iterating-through-an-array

**Review by azophy — APPROVED** [review]
LGTM. However I have a question:

Regarding the available templates, is there any documentation regarding the difference between each of them? Personally I'm not sure what the different category 6 is compared to the other. 

Maybe it could be linked in the Readme file or as comment inside the source code? Just a suggestion, in case there will be new team member in the future

**Comment by indraprasetya154** [comment]
> LGTM. However I have a question:
> 
> Regarding the available templates, is there any documentation regarding the difference between each of them? Personally I'm not sure what the different category 6 is compared to the other.
> 
> Maybe it could be linked in the Readme file or as comment inside the source code? Just a suggestion, in case there will be new team member in the future

Nice question.. basiclly we dont have any documentation like that for now on this repository 😢 
But the different between categories is position of QRCode and text on bottom pdf (footer section), Also we use category for grouping the file/document base on orientation and page size.

Thanks for the advice Mas @azophy the documentation on README file will be add later 😸

**Pull request merged**